_posts/2024-11-20-cockpit-329.md

```diff
@@ -0,0 +1,52 @@
+---
+title: Cockpit 329
+author: mvo
+date: '2024-11-20'
+tags: cockpit, files
+slug: cockpit-329
+category: release
+summary: Multi host warnings and File creation'
```

Comment:
```suggestion
summary: 'Improved warnings when connecting to remote hosts and Cockpit Files learns how to create new files'
```

Summary was missing quoting (which would break the build) and changed the summary too.